lib/video/video.js

```diff
@@ -104,14 +104,16 @@ class Video {
       );
     }
     if (this.options.video === true) {
-      await finetuneVideo(
-        this.videoDir,
-        this.videoPath,
-        this.index,
-        videoMetrics,
-        timingMetrics,
-        this.options
-      );
+      if (this.options.tuneVideo === true) {
```

Comment:
Like it is now this will have other complications? By default we add the time and then also some metrics ontop of the video. If the orange is there, the timer is not insync. We also add the metrics in the video, when it happens.

Reply:
I'm not sure I understand the issue you're pointing to. However, I can remove this flag now that the issue I was having was resolved and that the original videos are now true originals.

Reply:
Sorry, I'll try to explain better: if we add a flag where you as a user can choose not to tune the video, the video will by default have overlay with a timer and the metrics. But that timer will not be correct (=matching the video), the same with the metric. If we add that functionality, I think we also need to make sure we do not add the timer and text for the metrics.

Reply:
Ah I see, so I'm confused about this because it looks like adding the timer is done within this `fineTuneVideo` method so I assumed that flag would skip adding the timer as well.

Reply:
I've since remove the flag though because the rename to copyFile change I made fixed the issue.

Reply:
I see I could be wrong, I think the video code can be rewritten so its easier to follow.